cmd/explaintest/r/access_path_selection.result

```diff
@@ -23,24 +23,3 @@ id	count	task	operator info
 IndexReader_11	1104.45	root	index:Selection_10
 └─Selection_10	1104.45	cop	lt(test.access_path_selection.b, 3)
   └─IndexScan_9	3323.33	cop	table:access_path_selection, index:a, b, range:[-inf,3), keep order:false, stats:pseudo
-CREATE TABLE `outdated_statistics` (
```

Comment:
Why remove it？

Reply:
Because this case is intended to test that planner would prefer idx_ab over idx_a, but it is hard to say whether double read using idx_ab is better than table scan or not. With the new concurrency factor, planner would prefer table scan, because there are only 6 rows in total.